src/conda_smithy.rst

```diff
@@ -0,0 +1,54 @@
+conda-smithy rerender
+=====================
+
+conda-forge  uses `conda-smithy` to re-render the feedstock and update the feedstock configuration.
+This action updates the `README`, CI configuration, the build matrix,
+fix problems with the feedstock configuration, etc.
+
+The bot issues re-rendering PRs every now and then,
+but sometimes the users may need to issue them manually instead waiting for the bot.
+Here is how, and when, to re-render.
+
+How to re-render?
+-----------------
+
+First make sure you have both:
+    - both `defaults` and `conda-forge` channels in your `.condarc` and no other channel;
+    - the `conda-forge` channel should be on **top** of `defaults` to ensure package consistency via channel preference;
+    - updated version of `conda` and `conda-build`.
+
+To ensure the channels are present and in the right order you can use:
+
+```shell
+conda config --add channels defaults --force
+conda config --add channels conda-forge --force
```

Comment:
Not sure what the intent of this statement is, but channels in the `.condarc` no longer effect the outcome of re-rendering in `conda-smithy` 2.0.0+.